spec/v0.4.1-draft/spec.md

```diff
@@ -629,12 +629,17 @@ It implies stronger semantics than a [set relation](#set-relation).
 
 A group is modeled as a relation with a list of its members.
 
-| Property  | JSON Type | OCIF Type   | Required     | Contents                  |
-| --------- | --------- | ----------- | ------------ | ------------------------- |
-| `members` | `array`   | [ID](#id)[] | **required** | IDs of members of the set |
+| Property  | JSON Type | OCIF Type   | Required     | Contents                  | Default |
+| --------- | --------- | ----------- | ------------ | ------------------------- | ------- |
+| `members` | `array`   | [ID](#id)[] | **required** | IDs of members of the set |         |
+| `node`    | `string`  | [ID](#id)   | optional     | ID of a visual node       |         |
 
 - **members**: A list of IDs of nodes (or relations, such as other groups) that are part of the group.
 
+- **node**: The ID of a visual node, that represents the relation visually.
+  - If an node is used to visually represent a group, the node should point back via its `relation` to this group relation.
```

Comment:
Interesting. Regular nodes don't have a `relation` property to act as a reference to a relevant `relation`. We special-cased the `arrow` node to have that property due to the prevalence of `arrow` having an associated `edge` relation.

Perhaps we should promote `relation` as an option property to the top-level `node`? @xamde

Reply:
Yes, than seems to be a good idea

Reply:
I have changed this PR to move "node" and "relation" up to their root objects.

Reply:
Great! I'll resolve merge conflicts on this one and get it merged in. Will this cause conflicts for you?

Reply:
Ok, got PR 36 in. Give it a final review here and merge it in to 0.4.1.

Reply:
The merge conflicts seem to be outdated, as we already removed the set relation